packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,11 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [68.2.0]
+
 ### Changed
 
-- Bump `@metamask/controller-utils` to `^11.10.0` ([#5935](https://github.com/MetaMask/core/pull/5935))
-- Add `getErc20Balances` function within `TokenBalancesController` to support fetching ERC-20 token balances for a given address and token list ([#5925](https://github.com/MetaMask/core/pull/5925))
-  - This modular service simplifies balance retrieval logic and can be reused across different parts of the controller
```

Comment:
IDK why this was added, these are already in the changelog section below in `68.1.0`

Reply:
Hmm.... now that I look at it again, I think this does actually belong in 68.2.0. It doesn't show up in 68.1.0: https://github.com/MetaMask/core/commits/%40metamask/assets-controllers%4068.1.0/. I think it was mistakenly added for 68.1.0: https://github.com/MetaMask/core/pull/5925/files#diff-ee47d03d53776b8dd530799a8047f5e32e36e35765620aeb50b294adc3339fabR33

Reply:
Fixed in https://github.com/MetaMask/core/pull/5978/commits/356157b883037ef5bb12bea7a0aee14681bfb58b